build/Rulesets/Roslyn.ruleset

```diff
@@ -1,8 +1,115 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Common diagnostic rules for all Roslyn projects"
-         Description="This file contains diagnostic settings used by all Roslyn projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the necessary adjustments." 
-         ToolsVersion="14.0">
-  <Rules AnalyzerId="Roslyn.Core" RuleNamespace="Roslyn.Core">
-    <Rule Id="AD0001" Action="Error" />
-  </Rules>
-</RuleSet>
+﻿<?xml version="1.0" encoding="utf-8"?> 
+<RuleSet Name="Common diagnostic rules to run during build for all shipping Roslyn projects" Description="This file contains diagnostic settings used by all Roslyn projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the neceessary adjustments." ToolsVersion="14.0"> 
+  <IncludeAll Action="Hidden" /> 
+  <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers"> 
+    <Rule Id="CA1000" Action="None" /> 
```

Comment:
I'd just keep everything enabled in Microsoft.ApiDesignGuidelines.Analyzers and System.Runtime.Analyzers, and turn them off only if it is too noisy or needs a code fix with fixall for efficient fix.